sdk/client/src/otel/user-interaction.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="zone.js" />
```

Comment:
What does this do? Fix some type errors? Is this why we needed to add zone.js as a dependency?

Reply:
had type errors after updating the otel packages - i think they were previously importing zone.js as a transitive dep, but that's no longer the case. this seems to make typescript understand that zone.js is a dependency, but i'm not too sure. if you have an idea of how to do this better, lmk